src/main/java/pro/sky/JavaSkyPro/controller/DepartmentController.java

```diff
@@ -0,0 +1,44 @@
+package pro.sky.JavaSkyPro.controller;
+
+import org.springframework.web.bind.annotation.*;
+import pro.sky.JavaSkyPro.model.Employee;
+import pro.sky.JavaSkyPro.service.DepartmentService;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/department")
+public class DepartmentController {
+    private final DepartmentService departmentService;
+
+    public DepartmentController(DepartmentService departmentService) {
+        this.departmentService = departmentService;
+    }
+
+    @GetMapping("/{id}/employees")
+    public List<Employee> employeesDepartment(@PathVariable Integer department) {
+        return departmentService.getEmployeesDepartment(department);
+    }
+
+    @GetMapping("/{id}/salary/sum")
```

Comment:
работать не будет, у тебя параметр пути называется id, но нет связи с переменной. Надо либо переменную назвать id, либо указать название параметра в аннотации
```suggestion
    public List<Employee> employeesDepartment(@PathVariable("id") Integer department) {
```